api4/schema.graphqls

```diff
@@ -185,6 +212,14 @@ type Session {
 	token: String!
 	createAt: Float!
 	expiresAt: Float!
+	lastActivityAt: Float!
+	deviceId: String!
+	roles: String!
+	isOAuth: Boolean!
+	expiredNotify: Boolean!
+	props: StringMap!
+	# Deliberately leaving out teamMembers for now.
```

Comment:
I don't think this is a "for now" thing. I'm pretty sure this is only used on the server.

I'm also realizing our client-side type definitions also have a bunch of fields which the server never returns

Reply:
Understood, changed the comment.